Botkit/Microsoft.BotKit/BotkitDialogWrapper.cs

```diff
@@ -1,51 +1,56 @@
-// Copyright(c) Microsoft Corporation.All rights reserved.
-// Licensed under the MIT License.
-
-using Microsoft.BotKit.Conversation;
-using Microsoft.Bot.Builder.Dialogs;
-using System;
+﻿// <copyright file="BotkitDialogWrapper.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
 
 namespace Microsoft.BotKit
 {
+    using System;
+    using Microsoft.Bot.Builder.Dialogs;
+    using Microsoft.BotKit.Conversation;
+
+    /// <summary>
+    ///  BotkitDialogWrapper class.
+    /// </summary>
     public class BotkitDialogWrapper
     {
         /// <summary>
-        /// An object containing variables and user responses from this conversation.
+        /// Initializes a new instance of the <see cref="BotkitDialogWrapper"/> class.
         /// </summary>
-        public Tuple<string, object> Vars { get; set; }
-
+        /// <param name="dialogContext">dialogContext for the BotkitDialogWrapper.</param>
+        /// <param name="botkitconvoStep">botkitconvoStep for the BotkitDialogWrapper.</param>
         public BotkitDialogWrapper(DialogContext dialogContext, IBotkitConversationStep botkitconvoStep)
         {
-
         }
 
+        /// <summary>
+        /// Gets or sets an object containing variables and user responses from this conversation.
+        /// </summary>
+        public Tuple<string, object> Vars { get; set; }
+
         /// <summary>
         /// Jump immediately to the first message in a different thread.
         /// </summary>
-        /// <param name="thread">Name of a thread</param>
-        public async void GotoThread(string thread)
```

Comment:
I would leave these **_async_** indicators. The methods are not yet implemented, but, when they do, they should be async.

Reply:
ACK. Done